### ISSUE TITLE
(Feature request) Previous note button

### ISSUE THREAD

**Issue by deajan** [zadam/trilium#4249]
### Describe feature

Hello,

Whenever I use trilium for my linux notes, I usually need to follow a link from one note to another note to get in depth for a subject.
Once I've finished reading the deeper note, I need to navigate back to the previous note.
Since notes link open in the same tab unless I hit CTRL when cliking on an internal link (which I forget to do), whenever I need to get back to the previous note, I end up needing to search it to get back.

It would be so nice to have a "previous" button, just like in an internet navigator.

### Additional Information

Thanks for the time you're putting into this project.
Trilium is a major time saver for me.
I'm an IT engineer who writes lots of howtos and linux / hardware cheatsheets.
I'm currently recommending this to some coworkers and pupils (I also teach IT and use Trilium to show things via projector and share notes with pupils).

I've also already sponsored you personnaly in the past.
If I can get some receipt for being a sponsor, my company could chip in a bit to show some support.

My ultimate dream is that Trilium would end up being multiuser, since this would make it a major killer app for alot of engineering companies that use wikis as of today.

Best regards.

**Comment by rauenzi** [comment]
There are custom widgets that add things like navigation buttons. One such example is https://github.com/rauenzi/Trilium-Breadcrumbs (see below) maybe that would be a good stop-gap for you until it becomes a native feature.

https://user-images.githubusercontent.com/6865942/261645406-e3578d3d-78d9-4c32-84d6-063023a02f5f.mp4

**Comment by deajan** [comment]
Indeed, I've searched in keyboard shortcuts before, but I used "prev" in the search bar.
So yes, ALT + LEFT|RIGHT works great and will do me a great favor, thanks ;)

I still think small prev / next buttons would be useful for newcommers and people like me ;)

**Comment by zadam** [comment]
These buttons are in fact available in the "Configure Launchpad":

![](https://github.com/zadam/trilium/assets/617641/5dd26dc0-ee32-47eb-94f9-f372621c1879)

**Comment by deajan** [comment]
Thanks, indeed found them ;)

**Comment by zadam** [comment]
Sorry, I didn't read to the end.

> If I can get some receipt for being a sponsor, my company could chip in a bit to show some support.

TBH I don't know what kind of receipt should that be?

Also, even though I'm grateful for any sponsorship, the extra money can't really buy extra time I can dedicate to Trilium or even to specific features.

Recently there has been an influx of talented contributors submitting PRs, maybe we could set up some bounty system covered by GitHub/PayPal donations for desirable features. Although multi-user would be a tough nut (possibly impossible to crack).

**Comment by deajan** [comment]
> TBH I don't know what kind of receipt should that be?

If my company spends money into Trilium, I need some invoice or whatever in order for the accountants to be happy ;)

> Also, even though I'm grateful for any sponsorship, the extra money can't really buy extra time I can dedicate to Trilium or even to specific features.

Don't get me wrong, I don't want to sponsor in trade for new features, I just want to let you know your work is appreciated, in it's current state ^^

The multiuser feature I am talking about is a wet dream, I've read your comments about the subject in the issues before.
But I'm still hoping for this to happen once Trilium has reached some kind of v1.0 level.

Still, I've seen the multiple PRs in the last weeks and you're right, a bounty system would probably be a good idea.

**Comment by deajan** [comment]
@rauenzi I've totally overlooked your comment, and the plugin you have totally looks like what I'd expect in terms of UX.
Do you pledge to update the plugin for new trilium releases ?

**Comment by rauenzi** [comment]
I definitely will be, I use it myself!

**Comment by deajan** [comment]
@rauenzi Works like a charm ;)
